Derived address is wrong

marcotradenet opened the pull request.

zenit70 reviewed (commented):
@marcotradenet 

I understand that you have solved this simply by updating the libraries, is that correct?

marcotradenet commented:
> @marcotradenet
> 
> I understand that you have solved this simply by updating the libraries, is that correct?

I changed the library for bip32 and added the test for the mnemonic derivation that did not return the correct address

pasqenr reviewed (approved):
Looks great to me.
